packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -82,30 +75,30 @@ export const platformOptions: PreviewQuickPickItem[] = [
     defaultTargetName: ''
   },
   {
-    label: nls.localize('force_lightning_lwc_android_label'),
-    detail: nls.localize('force_lightning_lwc_android_description'),
+    label: nls.localize('force_lightning_lwc_preview_android_label'),
+    detail: nls.localize('force_lightning_lwc_preview_android_description'),
```

Comment:
the keys for our messages were a mix of `force_lightning_lwc_...` and `force_lightning_lwc_preview_...` so I changed all of our messages to use `force_lightning_lwc_preview_...` for consistency.